src/admin/scripts/indexCollections.js

```diff
@@ -55,6 +57,8 @@ async function main() {
  * Command: node -r dotenv/config src/admin/scripts/indexCollections --collections=Patient_4_0_0 --drop
  * Command: node -r dotenv/config src/admin/scripts/indexCollections --collections=all --drop
  * Command: node -r dotenv/config src/admin/scripts/indexCollections --synchronize
+ * Command: node -r dotenv/config src/admin/scripts/indexCollections --missingIndexesOnly
+ * Command: node -r dotenv/config src/admin/scripts/indexCollections --extraIndexesOnly
```

Comment:
update these flags and corresponding variables as
--addMissingIndexesOnly
--dropExtraIndexesOnly